python-checks/src/test/resources/checks/useOfOverrideOrOverloadDecorator.py

```diff
@@ -0,0 +1,21 @@
+from typing import Any
+
+class Class(UnknownParent):
```

Comment:
Probably here we could name this class `ClassUsingUnknownOverloadSymbols` or something similar to clearly express what the test is doing.